app/controllers/reports_controller.rb

```diff
@@ -4,9 +4,13 @@ class ReportsController < ApplicationController
   require_permission :can_view_reports?
 
   def index
-    @org_by_county = {}
-    Organization.counties.sort_by { |county| (county.presence || "no county").downcase }.each do |county_name|
-      @org_by_county[county_name.presence || "No County"] = Organization.where(county: county_name)
+    # raise params.inspect
+    @report = params[:report]
+    if @report == "value_by_county"
```

Comment:
Looking at the views, it seems more appropriate at this point to have 2 actions, since you are essentially disguising 2 actions as 1 here... they both have separate data and separate views.